applications/surgical_scene_recon/README.md

```diff
@@ -0,0 +1,652 @@
+   - Visit: https://github.com/med-air/EndoNeRF
+   - Download and extract the `pulling_soft_tissues` dataset
+   - Place it in: `${DATA_PATH}/EndoNeRF/pulling/`
+
+3. **Build Docker Image** (one-time, ~10-15 minutes)
+   ```bash
+   cd ${HOLOHUB_PATH}
+   docker build -f applications/surgical_scene_recon/Dockerfile -t surgical_scene_recon:latest .
+   ```
+
+4. **Run Complete Workflow** (RECOMMENDED - automated script)
+   ```bash
+   cd ${HOLOHUB_PATH}/applications/surgical_scene_recon
+   ./run_complete_docker.sh production
+   ```
+   
+   This automated script handles training + rendering in one command!
+   See [Complete Workflow Script](#complete-workflow-script-automated) section for details.
+
+   **OR** manually train and then run inference separately:
+   
+   a. Train the model:
+   ```bash
+   cd ${HOLOHUB_PATH}
+   docker run --rm --gpus all \
+     -v ${HOLOHUB_PATH}:/workspace/holohub \
+     -v ${DATA_PATH}:/workspace/data \
+     -w /workspace/holohub/applications/surgical_scene_recon \
+     surgical_scene_recon:latest \
+     python run_surgical_recon.py \
+       --mode train \
+       --data_dir /workspace/data/EndoNeRF/pulling \
+       --output_dir output/production_training \
+       --training_iterations 2000 \
+       --coarse_iterations 200
+   ```
+   
+   b. Run production inference:
+   ```bash
+   cd ${HOLOHUB_PATH}
+   docker run --rm --gpus all \
+     -e DISPLAY=$DISPLAY \
+     -v /tmp/.X11-unix:/tmp/.X11-unix \
+     -v ${HOLOHUB_PATH}:/workspace/holohub \
+     -v ${DATA_PATH}:/workspace/data \
+     -w /workspace/holohub/applications/surgical_scene_recon \
+     surgical_scene_recon:latest \
+     python run_surgical_recon.py \
+       --mode inference \
+       --data_dir /workspace/data/EndoNeRF/pulling \
+       --checkpoint /workspace/holohub/applications/surgical_scene_recon/output/production_training/ckpts/fine_best_psnr.pt
+   ```
+
+---
+
+## Key Features
+
+- ✅ **Real-time Visualization:** Stream surgical scene reconstruction at >10 FPS
+- ✅ **Temporal Deformation:** Accurate per-frame tissue modeling
+- ✅ **Tool Removal:** Tissue-only reconstruction mode (instruments excluded)
+- ✅ **Two Operation Modes:** Inference-only OR Train-then-render 🆕
+- ✅ **Integrated Training:** On-the-fly model training from streamed data 🆕
+- ✅ **Two Rendering Modes:** Static (fast) or Dynamic (high-quality)
+- ✅ **Holoviz Integration:** Interactive 3D visualization
+- ✅ **Production Ready:** Tested and optimized pipeline
+
+### What It Does
+
+**Input:** EndoNeRF surgical dataset (RGB images + camera poses + tool masks)  
+**Process:** Gaussian Splatting with temporal deformation network  
+**Output:** Real-time 3D tissue reconstruction without surgical instruments
+
+### Operation Modes
+
+1. **Inference-Only Mode:** Use pre-trained checkpoint for immediate rendering
+2. **Train-Then-Render Mode:** 🆕 Collect data → Train model → Render with trained checkpoint
+
+---
+
+## Quick Start
```

Comment:
Could you please bring the Quick Start to the top, maybe right after dataset? Thanks

Reply:
Done